radis/io/dbmanager.py

```diff
@@ -514,6 +516,27 @@ def get_nrows(self, local_file):
             raise ValueError(engine)
         return nrows
 
+    def get_columns(self, local_file):
+        """Get all columns (without loading all Dataframe"""
+        engine = self.engine
+        local_file = expanduser(local_file)
+        if engine == "vaex":
+            import vaex
+
+            # by default vaex does not load everything
+            df = vaex.open(local_file)
+            columns = df.columns
+            df.close()
```

Comment:
Note : you can use a with() to avoid the close() : 
```suggestion
            with vaex.open(local_file) as df:
                columns = df.columns
```

Reply:
Hi @erwan seems like with statement has `open` function associated with it, thus it overrides it when we are calling it,  thus performing different function which gives rise to error:
`Attribute Error:  __enter__ is not defined`
and also using different read-write operations it gives following error:
`ValueError: Unrecognized file extension. Please use .arrow, .hdf5, .parquet, .fits, or .csv to export to the particular file format.`

So I guess sticking with current implementation will be good.

Reply:
ok, no problem